ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -106,6 +107,7 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      ENABLE_OTA_TEST: OCP-27986
     test:
     - chain: openshift-upgrade-qe-sanity-disconnected
```

Comment:
Should we add a skip condition if the target version is not signed? From [log](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-stable-4.13-upgrade-from-stable-4.12-azure-ipi-fullyprivate-proxy-f28/1723769588191072256), it seems not signed always even is stable-to-stable upgrade.
```
Seem like registry.build01.ci.openshift.org/ci-op-trwh6vvv/release@sha256:f323afe3e80d44da7d0caaed8c353fedef37e5fd5e7f765e8440be86f546cba0 is not signed
You're updating to an unsigned images, you must override the verification using --force flag
warning: The requested upgrade image is not one of the available updates.You have used --allow-explicit-upgrade for the update to proceed anyway
warning: --force overrides cluster verification of your supplied release image and waives any update precondition failures.
```

Reply:
Ideally, I think all the stable builds should be signed, so I even think once the OTA case is enable, I prefer to run some pre-check to ensure the payload is signed. That is also a requirement for such cases.

On my localhost, I run some pre-check, it shows me singed, I am not sure what happened in the job, later I will check more.
```
[root@preserve-jialiu ~]# TARGET=registry.build01.ci.openshift.org/ci-op-trwh6vvv/release@sha256:f323afe3e80d44da7d0caaed8c353fedef37e5fd5e7f765e8440be86f546cba0
[root@preserve-jialiu ~]# check_signed
registry.build01.ci.openshift.org/ci-op-trwh6vvv/release@sha256:f323afe3e80d44da7d0caaed8c353fedef37e5fd5e7f765e8440be86f546cba0 is signed
```

Reply:
> From [log](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-stable-4.13-upgrade-from-stable-4.12-azure-ipi-fullyprivate-proxy-f28/1723769588191072256), it seems not signed always even is stable-to-stable upgrade.

I checked another [log](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-stable-4.13-upgrade-from-stable-4.12-azure-ipi-fullyprivate-proxy-f28/1701663851268804608/artifacts/azure-ipi-fullyprivate-proxy-f28/cucushift-upgrade-toimage/build-log.txt) , sounds like a flake.

Reply:
It seems the job triggered before the payload signed. 
```
https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/sha256%3Df323afe3e80d44da7d0caaed8c353fedef37e5fd5e7f765e8440be86f546cba0/
Nov 13 09:03:05 2023
```

Reply:
I added a retries logic to relief such edge cases.